timescaledb/tutorials/building-a-custom-timescaledb-dashboard.md

```diff
@@ -0,0 +1,326 @@
+  SELECT compress_chunk((c.chunk_schema ||'.' ||$1)::regclass)
+  FROM   timescaledb_information.chunks c
+  WHERE  NOT c.is_compressed
+  AND    c.chunk_name = $1 limit 1
+$$ LANGUAGE SQL VOLATILE;
+```
+
+Note that the function adds an extra where clause to not compress what is already compressed.
+
+![Tracking compress chunk mutation on Hasura cloud](https://assets.timescale.com/docs/images/tutorials/visualizing-compression/hasura-cloud-compress-chunk-mutation.png)
+
+#### Decompress chunk mutation
+
+We'll need a similar function for the decompression:
+
+```sql
+CREATE OR REPLACE FUNCTION decompress_chunk_named(varchar) returns setof compressed_chunk AS $$
+  SELECT decompress_chunk((c.chunk_schema ||'.' ||$1)::regclass)
+  FROM   timescaledb_information.chunks c
+  WHERE  c.is_compressed
+  AND    c.chunk_name = $1 limit 1
+$$ LANGUAGE SQL VOLATILE;
+```
+Now, the next step is to jump into the Hasura cloud and connect the database as a new data source.
+
+In the data panel, after setting up the PostgreSQL URI of your database, you can track each function as a query or mutation in a single click. Here is an example of compress_chunk_named function:
+
+In our case, the subscription goes to the chunks_with_compression, and here is what it looks like:
+
+You should also track decompress_chunk_named and compress_chunk_named as GQL mutations with a single argument.
+
+## Build the front-end visualization
+
+You can consult [the GitHub repo][repo-example] for the full code of our front-end application. The front-end application will connect to the Hasura GraphQL layer we created earlier, which, in turn, connects to our TimescaleDB database to retrieve information about chunks and compression status. The front-end application will then render the "dots" for our visualization.
+
+As a summary, in the front-end, we will do the following:
+
+1. Subscribe to the API via GraphQL
+2. Create an SVG component
+3. Iterate over all the chunks and add circles in the previous component
+4. Style the circle and add events to interact with it
+
+## Summary
+
+TimescaleDB is a powerful relational database for time-series data. But TimescaleDB is also built on top of PostgreSQL, the fastest growing database in the world. As a result, you can not only leverage the full spectrum of tools and dashboards available for PostgreSQL, but you can also build your own custom tools and dashboards to connect to PostgreSQL.
+
+The easiest way to get a TimescaleDB instance is to [try for free][timescale-signup] using our hosted service. You can also [download TimescaleDB for free][timescale-install] and run locally or in your own cloud infrastructure.
+
+You can get the full code for this project from [this GitHub repo][repo-example].
+
+We hope you find new ways to explore your data and make your decisions smarter and data-driven. If you get any interesting results or have any questions about this tutorial, drop us a line on our [community Slack channel][timescale-slack].
```

Comment:
```suggestion
You can use TimescaleDB to build your own custom tools and dashboards to connect to PostgreSQL. Get a TimescaleDB instance to [try for free][timescale-signup] using our hosted service. You can also [download TimescaleDB for free][timescale-install] and run it locally or in your own cloud infrastructure.

You can get the full code for this project from [this GitHub repo][repo-example].

We hope you find new ways to explore your data and make your decisions smarter and data-driven. If you get any interesting results or have any questions about this tutorial, drop us a line on our [community Slack channel][timescale-slack].
```

This doesn't read like a summary, more of a conclusion?

Reply:
Yes, you're right. When we initially build it, it was a blog post, but now we need to focus more on the Summary perspective. I'll be reworking on this part.

Reply:
I reworked it here, please review it again 🙇